### PR TITLE
OP-1218 Enable products medicals soft deletion

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1218.

### CODE REVIEW

File: src/main/java/org/isf/medicals/model/Medical.java [hunk 1]
```diff
@@ -21,6 +21,10 @@
  */
 package org.isf.medicals.model;
 
+import org.isf.medtype.model.MedicalType;
+import org.isf.utils.db.Auditable;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicals/model/Medical.java]: The import order defined in both Eclipse and IntelliJ has `jakarta` coming before `org`

**Review by dbmalkovsky — COMMENTED** [review]
Left a comment about import order; see the .ide-settings for Eclipse and IntelliJ that both have them different than what is in Medical.java

File: src/main/java/org/isf/medicals/model/Medical.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/medicals/model/Medical.java]: fixed, thanks!

**Review by dbmalkovsky — APPROVED** [review]
LGTM

**Pull request merged**